crates/ruff_python_parser/src/snapshots/ruff_python_parser__string__tests__backspace_alias.snap

```diff
@@ -1,19 +1,27 @@
 ---
 source: crates/ruff_python_parser/src/string.rs
 expression: suite
-snapshot_kind: text
 ---
 [
     Expr(
         StmtExpr {
+            node_index: NodeIndex(
+                "_",
+            ),
```

Comment:
Should we generate a custom `Debug` implementation for the nodes to avoid including this field in the snapshot? Or, is it required that we include this field?

Reply:
I don't really see the benefit of removing this field, we may end up setting it in the parser later. I updated the debug implementation of `NodeIndex` to avoid the extra whitespace.